### PR TITLE
BUG-794750 - Update Clustering Service versions

### PR DESCRIPTION
**Pull request by kingakowalska1**
Updated Clustering Service Docker images versions to the latest available.

### CODE REVIEW

**Review by hsomu — APPROVED** [review]

File: charts/pega/README.md [hunk 1]
```diff
@@ -1081,8 +1081,8 @@ The default and recommended deployment strategy for Hazelcast is client-server,
 Pega Infinity version   | Clustering Service version    |    Description
 ---                     | ---                           | ---
 < 8.6                   | NA                            | Clustering Service is not supported for releases 8.5 or below 
-\>= 8.6 && < 8.8         | \= 1.0.4                     | Pega Infinity 8.6 and later upto 8.7.x supports using a Pega-provided `platform-services/clustering-service` Docker Image that is tagged with version 1.0.3 or later patch version of clustering service. 
-\>= 8.8                 | \= 1.3.2                     | Pega Infinity 8.8 and later supports using a Pega-provided `platform-services/clustering-service` Docker Image that is tagged with version 1.3.0 or later patch version of clustering service. 
+\>= 8.6 && < 8.8         | \= 1.0.5                     | Pega Infinity 8.6 and later upto 8.7.x supports using a Pega-provided `platform-services/clustering-service` Docker Image that is tagged with version 1.0.3 or later patch version of clustering service.
```
Comment (taz-pega-work) [inline charts/pega/README.md:1084]: let's rephrase?>
Pega Infinity 8.6.x and 8.7.x supports using a Pega-provided `platform-services/clustering-service` Docker image that provides clustering service version 1.0.3 or later.
Pega Infinity 8.8 and later supports using a Pega-provided `platform-services/clustering-service` Docker image that provides clustering service version 1.3.0 or later.

**Review by taz-pega-work — COMMENTED** [review]
on minor suggestion to simplify the statement

File: charts/pega/README.md (hunk 1 above)
Comment (kingakowalska1) [reply charts/pega/README.md]: Good idea. :) Done.

**Review by taz-pega-work — APPROVED** [review]
thanks for collaborating!

**Review by hsomu — APPROVED** [review]

**Pull request merged**